src/test/java/ua/javaPro/berezhnoy/hillelHomeWork3/StringSymbolTest.java

```diff
@@ -0,0 +1,44 @@
+package ua.javaPro.berezhnoy.hillelHomeWork3;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+class StringSymbolTest {
+
+    @Test
+    void findSymbolOccurance() {
+        Assertions.assertEquals(1, StringSymbol.findSymbolOccurance("ad", 'd'));
+    }
+
+
+    @Test
+    void findWordPosition() {
+        Assertions.assertEquals(1, StringSymbol.findWordPosition("stany", "tany"));
+        Assertions.assertNotEquals(-1, StringSymbol.findWordPosition("stany", "tany"));
```

Comment:
```suggestion
        Assertions.assertEquals(-1, StringSymbol.findWordPosition("stany", "yyy"));
```
так було б зрозуміліше, оскільки в попередньому тесті ти вже перевірив, що  `findWordPosition("stany", "tany") == 1`